templates/about.hbs

```diff
@@ -32,7 +32,9 @@
             if it's okay to publicly discuss, or
             <a href="mailto:release@rust-lang.org">email the release team</a>
             if you'd prefer to be more discreet. We are happy to remove or
-            make edits where appropriate.
+            make edits where appropriate. You can also submit a pull request to
+            update the <code>.mailmap</code> files in the repositories you've
+            contributed to.
```

Comment:
Technically today only the `rust-lang/rust` .mailmap is actually used (but is used for all repositories). Can you update this snippet to indicate that?

Reply:
Done.